databricks_cli/sdk/api_client.py

```diff
@@ -106,5 +106,5 @@ def perform_query(self, method, path, data = {}, headers = None):
         try:
             resp.raise_for_status()
         except requests.exceptions.HTTPError as e:
-            raise e
+            raise requests.exceptions.HTTPError(resp.content, e)
```

Comment:
How does this work? Does HTTPError handle the arguments or does IOError do it?

Also, could we just use `e.response` where ever this exception is caught again? Not sure what the purpose of throwing this exception again is anyways actually.